packages/aws-cdk/test/integ/test-cli-framework.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
The names of these scripts are not intuitive to me. I would make these a bit more explicit and use the "regression" terminology. Something like:

- `regression-against-??.sh`

Reply:
Yeah I like it.